src/components/pages/UserPage/UserPage.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PageLayout from '<templates>/PageLayout/PageLayout';
+import headerMapper from '<helpers>/headerMapper';
+import displayLoader from '<helpers>/displayLoader';
+import FlexContainer from '<atoms>/layouts/FlexContainer/FlexContainer';
+import Image from '<atoms>/Image/Image';
+import ImageUrl from '<image>/homepage.png';
+import items from './navItems';
+import Title from '<atoms>/Title/Title';
+import SearchBar from '<molecules>/SearchBar/SearchBar';
+
+const UserPage = () => {
+  const navItems = headerMapper(items);
+
+  return (
+    <PageLayout navItems={navItems} callback={displayLoader}>
+      <FlexContainer justifyContent='space-between'>
+        <FlexContainer flexDirection='row'>
+          <FlexContainer
```

Comment:
i think you should use the already existing image component since it is being reused.